code/modules/surgery/helpers.dm

```diff
@@ -98,17 +102,14 @@
 		else
 			to_chat(user, "<span class='warning'>You need to hold a [is_robotic ? "screwdriver" : "cautery"] in your inactive hand to stop [M]'s surgery!</span>")
 
-/proc/get_location_modifier(mob/M)
+/proc/can_operate(mob/M)
 	var/turf/T = get_turf(M)
 	if(locate(/obj/structure/table/optable, T))
-		return 1
+		return TRUE
 	else if(locate(/obj/structure/table, T))
-		return 0.8
+		return TRUE

```

Comment:
Does this remove the modifier for ghetto surgery then?

Reply:
this proc is actually unused in tg and has been for a while. Ghetto surgery never had a modifier.

Reply:
this is a bug I need to fix